torch/distributed/pipelining/PipelineStage.py

```diff
@@ -543,16 +549,19 @@ def forward_one_chunk(
 
     def backward_one_chunk(
         self,
+        bwd_chunk_id: int,
```

Comment:
This is kinda unrelated but:

In schedule we call the pieces of the batch, microbatches. While in stage we refer to them as chunks and here `chunk_id` instead of `mb_index`. There is some inconsistency in terms and Feng did bring up since in the pipeline stage class he sees `num_chunks` which he thought was chunks of the model. 

What should we do for that? Just wanted to open up the discussion, if we decide changes can be in another PR